### ISSUE TITLE
WPF using Python Machine Learning Libraries

### ISSUE THREAD

**Issue by redcican** [QuantConnect/Lean#3520]
Hi, I was just wondering if I am in the right place?
Currently, I am working on a project to create a GUI by using WPF, we want to utilize the majority of Python Machine Leaning Libraries (Scikit-Learn, TensorFlow, Pandas, Numpy). So my question is: can I take advantage from LEAN?

**Comment by AlexCatarino** [comment]
Hi @redcican,
Actually, this question should be sent over to [QuantConnect Support](support@quantconnect.com).
We have been working on a GUI using [Electron](https://electronjs.org/) that it will emulate [QuantConnect Algorithm Lab](https://www.quantconnect.com/terminal/).

It is not clear how the WPF app will work. Do you want to use it to connect to QuantConnect.com or will it run Lean locally?

**Comment by redcican** [comment]
Hi, actually it should run locally. Since I have been looking for an appropriate way to solve my question for a long time. I thought that LEAN may help me.

**Comment by AlexCatarino** [comment]
Please contact support@quantconnect.com with more details. I don't know how we can help you with the information you gave us so far.
